vnext/Microsoft.ReactNative/ReactNativeHost.h

```diff
@@ -35,7 +34,6 @@ struct ReactNativeHost : ReactNativeHostT<ReactNativeHost> {
   Mso::CntPtr<Mso::React::IReactHost> m_reactHost;
 
   ReactNative::ReactInstanceSettings m_instanceSettings{nullptr};
```

Comment:
>m_instanceSettings [](start = 37, length = 18)

For my education: What do these settings now apply to?

Reply:
They apply for the ReactInstance used internally. We just do not expose it through the ABI.

---
In reply to: [431495017](https://github.com/microsoft/react-native-windows/pull/5030#discussion_r431495017) [](ancestors = 431495017)